[2024-10-22T03:24:08.576+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-22T03:24:08.586+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: auto_training_crawling_data_okx.training_model manual__2024-10-22T03:23:46.737974+00:00 [queued]>
[2024-10-22T03:24:08.592+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: auto_training_crawling_data_okx.training_model manual__2024-10-22T03:23:46.737974+00:00 [queued]>
[2024-10-22T03:24:08.592+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-22T03:24:08.610+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): training_model> on 2024-10-22 03:23:46.737974+00:00
[2024-10-22T03:24:08.622+0000] {standard_task_runner.py:72} INFO - Started process 378792 to run task
[2024-10-22T03:24:08.630+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'auto_training_crawling_data_okx', 'training_model', 'manual__2024-10-22T03:23:46.737974+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/full_pipeline.py', '--cfg-path', '/tmp/tmpf3glcnoi']
[2024-10-22T03:24:08.633+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask training_model
[2024-10-22T03:24:08.681+0000] {task_command.py:467} INFO - Running <TaskInstance: auto_training_crawling_data_okx.training_model manual__2024-10-22T03:23:46.737974+00:00 [running]> on host ecs-gpu
[2024-10-22T03:24:08.759+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='auto_training_crawling_data_okx' AIRFLOW_CTX_TASK_ID='training_model' AIRFLOW_CTX_EXECUTION_DATE='2024-10-22T03:23:46.737974+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-22T03:23:46.737974+00:00'
[2024-10-22T03:24:08.760+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-22T03:24:08.816+0000] {logging_mixin.py:190} INFO - Using device: cuda:0
[2024-10-22T03:24:08.821+0000] {logging_mixin.py:190} INFO - Lỗi trong load_data: could not convert string to float: '2024-10-22 00:00:00'
[2024-10-22T03:24:08.821+0000] {logging_mixin.py:190} INFO - Lỗi trong training_transformer: could not convert string to float: '2024-10-22 00:00:00'
[2024-10-22T03:24:08.822+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/airflow/dags/full_pipeline.py", line 142, in training_transformer
    raise e
  File "/root/airflow/dags/full_pipeline.py", line 61, in training_transformer
    train_dataset, val_dataset, test_dataset = load_data(df)
  File "/root/airflow/dags/full_pipeline.py", line 46, in load_data
    raise e
  File "/root/airflow/dags/full_pipeline.py", line 25, in load_data
    df = scaler.fit_transform(df)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_set_output.py", line 157, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 916, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/root/airflow/dags/scaler.py", line 34, in fit
    self.scaler.fit(X)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 435, in fit
    return self.partial_fit(X, y)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 1152, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 473, in partial_fit
    X = self._validate_data(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 605, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/validation.py", line 915, in check_array
    array = _asarray_with_order(array, order=order, dtype=dtype, xp=xp)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_array_api.py", line 380, in _asarray_with_order
    array = numpy.asarray(array, order=order, dtype=dtype)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/pandas/core/generic.py", line 1998, in __array__
    arr = np.asarray(values, dtype=dtype)
ValueError: could not convert string to float: '2024-10-22 00:00:00'
[2024-10-22T03:24:08.833+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=auto_training_crawling_data_okx, task_id=training_model, run_id=manual__2024-10-22T03:23:46.737974+00:00, execution_date=20241022T032346, start_date=20241022T032408, end_date=20241022T032408
[2024-10-22T03:24:09.136+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-22T03:24:09.137+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 50 for task training_model (could not convert string to float: '2024-10-22 00:00:00'; 378792)
Traceback (most recent call last):
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/root/airflow/dags/full_pipeline.py", line 142, in training_transformer
    raise e
  File "/root/airflow/dags/full_pipeline.py", line 61, in training_transformer
    train_dataset, val_dataset, test_dataset = load_data(df)
  File "/root/airflow/dags/full_pipeline.py", line 46, in load_data
    raise e
  File "/root/airflow/dags/full_pipeline.py", line 25, in load_data
    df = scaler.fit_transform(df)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_set_output.py", line 157, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 916, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/root/airflow/dags/scaler.py", line 34, in fit
    self.scaler.fit(X)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 435, in fit
    return self.partial_fit(X, y)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 1152, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/preprocessing/_data.py", line 473, in partial_fit
    X = self._validate_data(
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/base.py", line 605, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/validation.py", line 915, in check_array
    array = _asarray_with_order(array, order=order, dtype=dtype, xp=xp)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/sklearn/utils/_array_api.py", line 380, in _asarray_with_order
    array = numpy.asarray(array, order=order, dtype=dtype)
  File "/root/miniconda3/envs/project/lib/python3.8/site-packages/pandas/core/generic.py", line 1998, in __array__
    arr = np.asarray(values, dtype=dtype)
ValueError: could not convert string to float: '2024-10-22 00:00:00'
[2024-10-22T03:24:09.203+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-22T03:24:09.222+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-22T03:24:09.223+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
